src/Dots.spec.js

```diff
@@ -48,3 +48,14 @@ test('onDotClick handler', () => {
   )
   expect(tree).toMatchSnapshot()
 })
+
+test('custom dot color', () => {
+  const tree = renderer.create(
+    <ThemedDots
+      count={4}
+      index={0}
+      dotColor='red'
+    />
+  )
+  expect(tree).toMatchSnapshot()
```

Comment:
It would be better if this test would actually check if the dots have the specified color.

Reply:
Technically the snapshot testing would? If you'd like an explicit test, is there a way to do it without having to set up enzyme in the test environment?

Reply:
You're right, the snapshot test checks this already. :+1: